cdap-docs/admin-manual/source/operations/cdap-ui.rst

```diff
@@ -10,7 +10,7 @@ CDAP UI
 =======
 
 The **CDAP UI** is available for deploying, querying and managing the Cask Data
-Application Platform in all modes of CDAP except as an 
+Application Platform in all modes of CDAP except as an
```

Comment:
This statement seems wrong.. the UI doesn't really help to deploy CDAP. Shall we replace with:

CDAP contains a web UI for deploying and managing applications and datasets. In addition, it also contains UIs for the data pipeline and data preparation applications.